lib/spack/spack/test/cmd/bootstrap.py

```diff
@@ -38,6 +39,8 @@ def test_root_get_and_set(mutable_config, scope):
 
     _bootstrap('root', path, *scope_args)
     out = _bootstrap('root', *scope_args, output=str)
+    if sys.platform == 'win32':
+        out = out.replace("\\", "/")
```

Comment:
For those cases where doing this is necessary (this seems like one of those) - I think it might be helpful to create a `convert_to_posix_path` function: it would only be one line, but the intent would be slightly clearer IMO.

Reply:
Yeah, that's probably something I should have thought to include in the path PR. I'll add it now.